crates/rolldown_common/src/types/import_record.rs

```diff
@@ -52,23 +52,35 @@ pub struct RawImportRecord {
   pub kind: ImportKind,
   /// See [ImportRecord] for more details.
   pub namespace_ref: SymbolRef,
-  /// See [ImportRecord] for more details.
-  pub contains_import_star: bool,
-  /// See [ImportRecord] for more details.
-  pub contains_import_default: bool,
-  /// See [ImportRecord] for more details.
-  pub is_plain_import: bool,
+  pub module_request_start: u32,
+  pub import_record_meta: ImportRecordMeta,
```

Comment:
```suggestion
  pub meta: ImportRecordMeta,
```

Reply:
If you think this might cause confusing with considering the `meta` word used in link stage. You could call it `flags` too.